src/BizHawk.Client.EmuHawk/Extensions/ToolExtensions.cs

```diff
@@ -242,7 +242,10 @@ public static IEnumerable<ToolStripItem> MenuItems(this IMemoryDomains domains,
 				var item = new ToolStripMenuItem
 				{
 					Text = name,
-					Enabled = !(maxSize.HasValue && domain.Size > maxSize.Value),
+					// 0-length domains cause several problems, so they're disabled.

```

Comment:
We have many cores with conditional domains that only show in certain (undocumented) situations. 

I think better is to not add the memory domain at all, at the core level if it is 0 bytes.  

If for some reason that isn't feasible, they should just be filtered when grabbing a list of domains from the core.